doc/pagerank.adoc

```diff
@@ -48,20 +54,54 @@ CREATE (home)-[:LINKS]->(about)
 
 ----
 
-.Running algorithm and streaming results
+==== Running algorithm on a _directed_, _unweighted_ graph and streaming results
+
+.Loading the subset of the graph with labels and relationship type
 [source,cypher]
 ----
 
-CALL algo.pageRank.stream('Page', 'LINKS', {iterations:20, dampingFactor:0.85}) YIELD node, score
+CALL algo.pageRank.stream('Page', 'LINKS', {iterations:20, dampingFactor:0.85}) 
+YIELD node, score
 
 ----
 
-.Running algorithm and writing back results
```

Comment:
I am trying to find out how to tell people which of the four versions of (un)directed,(un)weighted graphs do our algos support. I think maybe the best way is to write a new section with each combination and a ticked checkbox of those that we support ?

Also remove too much information from the simple example.